### ISSUE TITLE
Feature request landscape and ordering

### ISSUE THREAD

**Issue by Lorestan00** [axel358/smartdock#70]
Would be great to have the following 

Landscape mode option to have more apps showing to better use space

Ability to order recent apps on the left hand side of the dock as well as the right

Handle transparency option. Hide the handle abit

Thanks

**Comment by axel358** [comment]
Landscape mode is just a matter of locking your device orientation to landscape. We'll be working on transparency for the handle for the next release. Not sure what u mean by ordering

**Comment by Lorestan00** [comment]
Sorry I've not explained myself in landscape it would be great to have additional apps showing i.e. an option in landscape to have say 10 apps make use of the extra space and in portrait 5 apps

With the order of recent apps in the dock at the moment show the most recently used to the right. it would be great to flip this around so they appear on the left hand side. As someone who holds their phone in their left hand it would be much easier to reach the most recent apps this way.

Great news on the transparency option

**Comment by KeronCyst** [comment]
Yes, I _strongly_ prefer having the most recent apps on the left, not the right, too.

**Comment by axel358** [comment]
Implemented in f056a0abc0cb630a2611179e65019f0aa6dfea72
